tailwind.config.ts

```diff
@@ -1,10 +1,12 @@
+import type { Config } from 'tailwindcss';
```

Comment:
Holding off on migrating to tailwind 4 — good news is that it supports plugins like typography (which we use) using a pure css syntax, so that's great